packages/manager/modules/pci/src/projects/project/users/users.service.js

```diff
@@ -14,6 +16,17 @@ export default class PciProjectsProjectUsersService {
     this.OvhApiCloudProject = OvhApiCloudProject;
     this.OvhApiCloudProjectRegion = OvhApiCloudProjectRegion;
     this.OvhApiCloudProjectUser = OvhApiCloudProjectUser;
+    this.ALPHA_CHARACTERS_REGEX = ALPHA_CHARACTERS_REGEX;
+  }
+
+  checkGlobalRegion(regions) {
+    let hasGlobalRegions = false;
+    forEach(regions, (region) => {
+      if (this.ALPHA_CHARACTERS_REGEX.test(region.id)) {
+        hasGlobalRegions = true;
+      }
+    });
```

Comment:
This could be a `some` 
```suggestion
    return some(regions, region => ALPHA_CHARACTERS_REGEX.test(region.id))
```